README.md

```diff
@@ -1,4 +1,6 @@
 # Benchmark-Goal-15-Coding-Exercises-round-2
+## Team Name: responsible-duck
+## Team Member(s): Jacklyne Bouier
 ## Description
 Benchmark goals are projects that are designed to be done solo. 
```

Comment:
I just added my Team name and members to the original goal specs . Specs should be the same. Also changed the title a bit.